pumps/graph_sql_aggregate_test.go

```diff
@@ -74,14 +74,17 @@ const (
 )
 
 func TestSqlGraphAggregatePump_Init(t *testing.T) {
+	if os.Getenv("TYK_TEST_POSTGRES") == "" {
+		t.Skip("Skipping test because TYK_TEST_POSTGRES environment variable is not set")
+	}
```

Comment:
what about if we move this to a helper function? in sink we have:

```
func ValidTestType(t *testing.T, storageType config.DBType) {
	t.Helper()
	config.LoadConfig("")

	if config.GetConfig().AnalyticsConfig.Type == storageType {
		return
	} else {
		t.Skip()
	}
}
```
wondering if we should something similar here

Reply:
I see down below you already have: `skipTestIfNoPostgres` lets use that

Reply:
I forgot to add skipTestIfNoPostgres there